CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- **[NEW]** Add `CarConvertibleIcon`, `CarEstateIcon`, `CarHatchbackIcon`, `CarMinivanIcon`, `CarMvpIcon`, `CarSaloonIcon`, `CarSuvIcon`, `CarVanIcon`
```

Comment:
I've let a comment on Figma, but _Estate_ is _Break_ in English and it should be reviewed
https://www.linguee.fr/anglais-francais/traduction/estate+car.html

Reply:
Renamed it.